### DISCUSSION TITLE
Relative paths

### DISCUSSION THREAD

**Discussion by ivanvinicius** [vercel/next.js#46576] [Help]
### Summary

Creating a new project we have to setup import alias

How can I skip this step and use relative paths instead?

**Comment by icyJoseph** [comment]
You can always just use relative paths, without having to do anything.

If you want to remove the possibility of import alias, you'd need to remove this from `tsconfig.json`:

```json
    "paths": {
      "@/*": ["./src/*"]
    }
```

Now, that may not be ideal. Other flags passed to the CLI have the `--no` prefix option , for example `--no-app-dir`, perhaps a PR can be created for this, but I think it is a no-op, because even if an import alias is forced, you can just ignore it and use relative paths.
